recipes/variant-effect-predictor/build.sh

```diff
@@ -29,3 +29,4 @@ rm -rf t/
 wget --no-check-certificate https://raw.githubusercontent.com/ensembl-variation/VEP_plugins/master/CADD.pm
 wget --no-check-certificate https://raw.githubusercontent.com/ensembl-variation/VEP_plugins/master/dbNSFP.pm
 wget --no-check-certificate https://raw.githubusercontent.com/konradjk/loftee/master/LoF.pm
```

Comment:
Ok, that's also a valid point. 
I'll just point the url's to to the latest commit on the master branch. The plugin url's can then be updated together with the vep version.